recipes/crow/all/conanfile.py

```diff
@@ -0,0 +1,46 @@
+from conans import ConanFile, tools, CMake
+import os
+
+required_conan_version = ">=1.28.0"
+
+class CrowConan(ConanFile):
+    name = "crow"
+    homepage = "https://github.com/ipkn/crow"
+    description = "Crow is C++ microframework for web. (inspired by Python Flask)"
+    topics = ("conan", "web", "microframework", "header-only")
+    url = "https://github.com/conan-io/conan-center-index"
+    settings = "os", "compiler", "arch", "build_type"
+    generators = "cmake", "cmake_find_package", "cmake_find_package_multi"
+    exports_sources = ["patches/*"]
+    license = "BSD3"
```

Comment:
https://spdx.org/licenses/BSD-3-Clause.html

```suggestion
    license = "BSD-3-Clause"
```